### ISSUE TITLE
Add a script to upload video to Youtube automatically

### ISSUE THREAD

**Issue by krishnanunnir** [krishnanunnir/reddit-compilation-maker#3]
It would be really convenient if we could do that and very cool too.

**Comment by jnash10** [comment]
Still open?
I'd like to tackle it.
It will be my first issue as the tag suggests, any tips?

**Comment by krishnanunnir** [comment]
Yes, it is open. Please feel free to work on it.
Just do your thing man. Ping me in case of any doubts or you require any help.

**Comment by jnash10** [comment]
Do we want people to go through the whole 'App creation' process on Google similar to how we did on reddit?
![apikey](https://user-images.githubusercontent.com/67156641/91936216-115eba80-ed0d-11ea-8a0e-2959d98f68d8.png)

**Comment by jnash10** [comment]
There does exist a CLI for similar purposes: https://github.com/tokland/youtube-upload
However, the user must create an app on Google similar to Reddit, but the Google process is more complicated(to generate client and service IDs)

Could you have a look at it and suggest which way to proceed?

**Comment by krishnanunnir** [comment]
Yes, I had a look at the API, it seems to be a bit more complicated, but try implementing it, there are tons of resources available. The official [documentation](https://developers.google.com/youtube/v3/guides/uploading_a_video) seems to be pretty good. 

I don't think we need something very heavy like the Youtube Upload tool for our use-case. 

I think the best way to go would be to build the python script for youtube upload rather than using a third-party library as it would help you learn a lot about handling APIs.

The code might look intimidating at first, but once you start building it, unit by unit, it will get simpler and simpler. If you have any issues please feel free to contact me.
